### PR TITLE
RuntimeException at WS client level must be managed as technical error.

### PR DESCRIPTION
**Pull request by cdeneux**
This PR completes PR #1011 that does not manage the RuntimeException occurring at web-service client level.

Such runtime exception can occur when the web-service endpoint is defined with an URL where the scheme part is unknown.

### CODE REVIEW

File: modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java [hunk 1]
```diff
@@ -97,23 +97,39 @@ public void testFaultManagement() throws Exception {
...
         webServiceMock.setTo(123456);
         try {
             runtimeService.startProcessInstanceByKey("webServiceInvocation");
         } catch (ActivitiException e) {
+            assertFalse("Exception processed as Business fault", e instanceof BpmnError);
             assertTrue(e.getCause() instanceof SoapFault);
         }
 
-        // Unexpected fault invoking the web-service, so not catched in the process definition
+        // Unexpected fault at ws-client side invoking the web-service, so not catched in the process definition
```
Comment (dbmalkovsky) [inline modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java]: Should read:
so not caught in the process definition.

File: modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java [hunk 2]
```diff
@@ -97,23 +97,39 @@ public void testFaultManagement() throws Exception {
...
             assertTrue(e.getCause() instanceof Fault);
         } finally {
             server.start();
         }
     }
 
+    @Deployment
+    public void testFaultManagementREatWSClientLevel() throws Exception {
+
+        // RuntimException occurring during processing of the web-service at ws-client side, so not catched in the
```
Comment (dbmalkovsky) [inline modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java]: Should read:
RuntimeException occurring during processing of the web-service at ws-client side, so not caught

File: modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java (hunk 1 above)
Comment (cdeneux) [inline modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java]: Yes, an unexpected fault returned by the WS client stack is a technical error and should not be caught in the process definition

File: modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java [hunk 3]
```diff
@@ -97,23 +97,39 @@ public void testFaultManagement() throws Exception {
         assertEquals(1, historicProcessInstanceWithExpectedFault.size());
         assertEquals("theEndWithError", historicProcessInstanceWithExpectedFault.get(0).getEndActivityId());
 
-        // Runtime exception occurring during processing of the web-service, so not catched in the process definition
+        // RuntimException occurring during processing of the web-service, so not catched in the process definition. The RuntimeException
```
Comment (cdeneux) [inline modules/activiti-cxf/src/test/java/org/activiti/engine/impl/webservice/WebServiceTaskTest.java]: Yes, sure

**Comment by vasiledirla** [comment]
Thanks @cdeneux !

**Pull request merged**